codec-http2/src/main/java/io/netty/handler/codec/http2/Http2ConnectionHandler.java

```diff
@@ -153,6 +134,10 @@ public void channelActive(ChannelHandlerContext ctx) throws Exception {
 
     @Override
     public void handlerAdded(ChannelHandlerContext ctx) throws Exception {
+        // Initialize the encoder and decoder.
+        encoder.lifecycleManager(this);
```

Comment:
@Scottmitch I didn't provide a get method for the `lifecycleListener` to this point, since it's not needed for anything.  I'm not convinced it's worth it just to handle this ... WDYT?